pnpm-lock.yaml

```diff
@@ -120,7 +120,7 @@ importers:
         version: 8.2.6(react-dom@18.3.1)(react@18.3.1)(storybook@8.2.6)(typescript@5.5.3)
       '@storybook/react-webpack5':
         specifier: 8.2.6
-        version: 8.2.6(esbuild@0.18.20)(react-dom@18.3.1)(react@18.3.1)(storybook@8.2.6)(typescript@5.5.3)(webpack-cli@5.1.4)
```

Comment:
I think we shouldn't see any changes here. If you discard them it should still be fine